sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -217,12 +217,17 @@ public String getEntityName() {
      *
      * @throws NullPointerException if {@code message} or {@code scheduledEnqueueTime} is {@code null}.
      */
-    public Mono<Long> schedule(ServiceBusMessage message, Instant scheduledEnqueueTime) {
+    public Mono<Long> scheduleMessage(ServiceBusMessage message, Instant scheduledEnqueueTime) {
         Objects.requireNonNull(message, "'message' cannot be null.");
         Objects.requireNonNull(scheduledEnqueueTime, "'scheduledEnqueueTime' cannot be null.");
 
-        //TODO (hemanttanwar): Implement session id feature.
-        return Mono.error(new IllegalStateException("Not implemented."));
+        return connectionProcessor
+            .flatMap(connection -> connection.getManagementNode(entityName))
+            .flatMap(x -> x.schedule(message, scheduledEnqueueTime))
+            .flatMap(scheduleSequenceNumber -> {
```

Comment:
.map is sufficient since this is not an asynchronous operation. Also replace usage of System.out with logger